src/Workspaces/Core/Portable/CodeFixes/FixAllOccurrences/FixAllContext.DiagnosticProvider.cs

```diff
@@ -24,8 +25,6 @@ public partial class FixAllContext
         /// </summary>
         public abstract class DiagnosticProvider
         {
-            internal virtual bool IsFixMultiple => false;
```

Comment:
Internal virtual/abstract members on public types are evil and indicate code smell. Might be worth having an analyzer to flag these.